motioneye/mjpgclient.py

```diff
@@ -40,8 +40,8 @@ class MjpgClient(IOStream):
     def __init__(self, camera_id, port, username, password, auth_mode):
         self._camera_id = camera_id
         self._port = port
-        self._username = (username or '').encode('utf8')
-        self._password = (password or '').encode('utf8')
+        self._username = username or ''
+        self._password = password or ''
```

Comment:
Generally trying to understand the effect of these changes:
- https://github.com/motioneye-project/motioneye/blob/0300187/motioneye/utils/__init__.py#L281-L282: `build_basic_header` in several functions below calls `base64.encodebytes` which requires bytes, not string, so isn't removing the `encode` here breaking those?
- Or was it bytes already in some cases? If so, we need to check how the client is initiated and assure consistency, then using `b''` as empty bytes string fallback.